definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js

```diff
@@ -58,96 +82,207 @@ declare module 'antd' {
     disabled?: boolean,
-    expandTrigger?: 'click' | 'hover',
+    expandTrigger?: "click" | "hover",
     notFoundContent?: string,
     onChange?: (values: string[]) => mixed,
     options: CascaderOption[],
     placeholder?: string,
     popupClassName?: string,
-    popupPlacement?: 'bottomLeft' | 'bottomRight' | 'topLeft' | 'topRight',
+    popupPlacement?: "bottomLeft" | "bottomRight" | "topLeft" | "topRight",
     popupVisible?: boolean,
     showSearch?: boolean,
-    size?: 'large' | 'default' | 'small',
+    size?: "large" | "default" | "small",
     value?: string[]
-  }
+  };
 
   declare export class Cascader extends React$Component<CascaderProps> {}
 
   declare export class Checkbox extends React$Component<{}> {}
 
   declare export class Col extends React$Component<{}> {}
 
-  declare export type CollapsePanelProps = {}
+  declare export type CollapsePanelProps = {};
 
   declare class CollapsePanel extends React$Component<CollapsePanelProps> {}
 
-  declare export type CollapseProps = {}
+  declare export type CollapseProps = {};
 
   declare export class Collapse extends React$Component<CollapseProps> {
-    static Panel: typeof CollapsePanel
+    static Panel: typeof CollapsePanel;
   }
 
   declare export class DatePicker extends React$Component<{}> {}
 
   declare export class Dropdown extends React$Component<{}> {}
 
-  declare export type FormProps = {}
+  declare type ValidationRule = {
+    enum?: string | string[],
+    len?: number,
+    max?: number,
+    message?: string,
+    min?: number,
+    pattern?: RegExp,
+    required?: boolean,
+    transform?: (value: mixed) => mixed,
+    type?: string,
+    validator?: (
+      rule: mixed,
+      value: mixed,
+      callback: mixed,
+      source?: mixed,
+      options?: mixed
+    ) => mixed,
+    whitespace?: boolean
+  };
+  declare type ValidateCallback = (erros: mixed, values: mixed) => void;
+  declare type GetFieldDecoratorOptions = {
+    exclusive?: boolean,
+    getValueFromEvent?: (...args: mixed[]) => mixed,
+    initialValue?: mixed,
+    normalize?: (value: mixed, prevValue: mixed, allValues: mixed) => mixed,
+    rules?: ValidationRule[],
+    trigger?: string,
+    validateFirst?: boolean,
+    validateTrigger?: string | string[],
+    valuePropName?: string
+  };
+
+  declare type WrappedFormUtils = {
+    getFieldDecorator(
+      id: string,
+      options?: GetFieldDecoratorOptions
+    ): (node: React$Node) => React$Node,
+    getFieldError(name: string): mixed[],
+    getFieldsError(names?: Array<string>): mixed,
+    getFieldsValue(fieldNames?: Array<string>): mixed,
+    getFieldValue(fieldName: string): mixed,
+    isFieldsTouched(names?: Array<string>): boolean,
+    isFieldTouched(name: string): boolean,
+    isFieldValidating(name: string): boolean,
+    resetFields(names?: Array<string>): void,
+    setFields(obj: Object): void,
+    setFieldsValue(obj: Object): void,
+    validateFields(callback: ValidateCallback): mixed,
```

Comment:
@ramsesz Is this the way to do same method, different attributes? Asking because it's not working for me - only takes the last `validateFields()` and won't stop complaining that `options` is missing when I only pass in a callback :(